tests/ui/test_ui_tools.py

```diff
@@ -233,8 +233,8 @@ def test_load_new_messages_mocked_log(self, mocker, msg_view,
                                                         anchor=0)
 
     @pytest.mark.parametrize("event, button, keypress", [
-        ("mouse press", 4, "up"),
-        ("mouse press", 5, "down"),
+        ("mouse press", 4, next(iter(keys_for_command('GO_UP')))),
+        ("mouse press", 5, next(iter(keys_for_command('GO_DOWN')))),
```

Comment:
Since these calls (or the `pop` I recommend elsewhere) are non-deterministic, we should probably take a different testing approach - we want to know that one of the valid up or down keypress keys was used, but that any should work. We could maybe parametrize further, or take another approach.

Reply:
I could only validate the first bit (one of the valid up or down keypress keys was used). Could you assist me with how to approach the second bit (but any should work)?

Reply:
To clarify, I mean by 'any should work', that any key extracted using pop should work, so you should be able to work through those in a test parametrization, or another similar approach.